File: 2728-sum-in-a-matrix/sum-in-a-matrix.py
class Solution:
    def matrixSum(self, nums: List[List[int]]) -> int:
        n = len(nums)
        m = len(nums[0])
        tot = 0
        for i in range(n):
            nums[i].sort(reverse=True)
        for i in range(m):
            maxi = 0
            for j in range(n):
                maxi = max(maxi, nums[j][i])
            tot += maxi
        return tot

File: 2728-sum-in-a-matrix/sum-in-a-matrix.java
class Solution {
    public int matrixSum(int[][] nums) {
        int n=nums.length;
        int m=nums[0].length;
        for(int i=0; i<n; i++){
            Arrays.sort(nums[i]);
        }
        int tot=0;
        for(int i=0; i<m; i++){
            int max=0;
            for(int j=0; j<n; j++){
                max=Math.max(max,nums[j][i]);
            }
            tot+=max;
        }
        return tot;
    }
}